packages/starlight/components/SidebarSublist.astro

```diff
@@ -43,6 +47,30 @@ interface Props {
     margin-top: 0.75rem;
   }
 
+  .sidebar-group-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-inline: var(--sl-sidebar-item-padding-inline);
+    cursor: pointer;
+    user-select: none;
+  }
+  
+  h2 {
+    padding-inline: unset
+  }
```

Comment:
Oh, I think we don’t need this. We can just edit the `h2` style on line 43 above to remove `padding-inline`!

Reply:
Sorry, I didn't see it earlier...